snmp/datadog_checks/snmp/data/default_profiles/chatsworth_pdu.yaml

```diff
@@ -497,6 +505,14 @@ metrics:
         column:
           OID: 1.3.6.1.4.1.30932.1.10.1.7.100.1.2
           name: cpiPduLockId
+      - tag: eas_status
+        column:
+          OID: 1.3.6.1.4.1.30932.1.10.1.7.100.1.3
+          name: cpiPduEasStatus
+        mapping:
+          0: inactive
+          1: ready
+          2: error
```

Comment:
1/ I think it might be better to completely create a new table with `cpiPduEas` as constant metric one.
And then add the tags.

The issue with the current table above is that it's going to create metric like `cpiPduEasStatus` tagged with `cpiPduEasStatus`, which is a bit strange since the metric value and metric tags are the same.

2/ If we are ok with 1/, beside tagging by `eas_status`, shall we also tag by the other status like cpiPduLockStatus and cpiPduDoorStatus

Reply:
Oh yes I see the issue, thanks for pointing out !

Reply:
Wait, isn't it the same as [here](https://github.com/DataDog/integrations-core/pull/15467/files#diff-005c5971c284116df6a241a5764f7788ef2e0179691a9fcfc5083c998db858e3R184-R188) ?

Reply:
@lizaerem I think it makes sense to use constant_value_one metrics with other numerical metrics.

But here where added to cpiPduDoorStatus/cpiPduLockStatus as "metrics"  instead of tags, since that the time, the constant_value_one feature didn't exist.